qemu/tests/cfg/nic_hotplug.cfg

```diff
@@ -104,7 +104,7 @@
         - with_iommu:
             only virtio_net
             only nic_virtio
-            netdev_extra_params_hotplug_nic1 = ",vhost=on,vhostforce=on"
+            netdev_extra_params_hotplug_nic1 = ",vhost=on"
```

Comment:
@PaulYuuu  Could you help check if this change is ok for ppc ? Thanks!

Reply:
vhostforce is not mandatory by this test case, it is related to MSI-X of guest. So I have not concerned of this change.